zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -400,141 +350,207 @@ impl<F: Field, const MAX_VERIF: usize> SignVerifyChip<F, MAX_VERIF> {
         })
     }
 
+    #[allow(clippy::too_many_arguments)]
+    fn assign_rlc_le(
+        &self,
+        config: &SignVerifyConfig,
+        ctx: &mut RegionCtx<F>,
+        chips: &ChipsRef<F, NUMBER_OF_LIMBS, BIT_LEN_LIMB>,
+        name: &str,
+        q_rlc: Selector,
+        challenge: Value<F>,
+        inputs_le: impl IntoIterator<Item = Term<F>>,
+    ) -> Result<AssignedCell<F, F>, Error> {
+        let zero = chips.main_gate.assign_constant(ctx, F::zero())?;
+        let columns = config.main_gate_config.advices();
+        let inputs_le = inputs_le.into_iter().collect_vec();
+        let inputs_be = iter::repeat_with(|| Term::assigned(zero.cell(), Value::known(F::zero())))
+            .take(Integer::next_multiple_of(&inputs_le.len(), &columns.len()) - inputs_le.len())
+            .chain(inputs_le.into_iter().rev())
+            .collect_vec();
+
+        let mut rlc = challenge.map(|_| F::zero());
```

Comment:
What are we trying to get from the expression `challenge.map(|_| F::zero())`? If it is `Value::known(F::zero())`, why dont we do so directly?

Reply:
Yeah using `Value::known(F::zero())` directly seems more straightforward, updated in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/803/commits/7f123a96ccda192a89f6775cf9cde4f650b65220.